### DISCUSSION TITLE
Upgrade to Github paid subscription?

### DISCUSSION THREAD

**Discussion by Ivanderkelen** [CORDEX-be2/ValEnsPy#66] [Admin]
Now, we are using the free version of Github. There are functionalities not available in the free version, from which we could benefit.  (see minutes meeting #50)

## Investigation of needs and prices
Github pricing plans: https://github.com/pricing, of which the intermediate **Team** sounds most interesting for us. 

### Interesting features of **Team** subscription plan

-	**Protected branches:** 
_“Enforce restrictions on how code branches are merged, including requiring reviews by selected collaborators, or allowing only specific contributors to work on a particular branch.”_
-	**Draft pull requests**
_Easily discuss and collaborate on pull requests before submitting to formal review._
-	**Pages and Wikis** 
_“Host documentation and simple websites for your project in a wiki format that contributors can easily edit either on the web or command line.”_

Additional features for us: Access to GitHub Codespaces, Mutliple reviewers in pull requests, Code owners, required reviewers, 2 GB of package storage (instead of 500MB). 

### Costs
Pricing for **Team**: $4/user/month. 
Currently (27/5/2024) we have 9 'seats' (active users). -> price per year: $396/year

**Comment by Ivanderkelen** [comment]
Added to the agenda for next meeting for further discussion. @kobebryant432 any additions/thoughts we should discuss too?

**Reply by kobebryant432** [reply]
This is linked to going public with the package.

**Comment by Ivanderkelen** [comment]
- [ ] Remove this discussion when going public!

**Reply by kobebryant432** [reply]
Good point! When going public we will have to scan the whole repo and paths as well.
